wasm_runtime/src/lib.rs

```diff
@@ -13,4 +13,4 @@ mod execution_ctx;
 mod wasm_engine;
 mod wasi_ctx;
 mod ffi_utils;
-mod c_api;
+pub mod c_api;
```

Comment:
I guess this is made public so that it can be used in `wasm_runtime/tests`. We can keep it private (so our test specific requirements do not permeate into the API), but instead of using `wasm_runtime/tests`, we can use `examples`. `examples` have access to non-public API's while they can be executed with `cargo run` too.

https://doc.rust-lang.org/cargo/reference/cargo-targets.html?highlight=examples#examples

Reply:
Yes, it's needed for the Integration Tests (`tests/integration_tests.rs`).
And on the other hand, it makes sense since it's actually the module that provides the API to the library.